packages/jest-matchers/src/index.js

```diff
@@ -149,6 +152,8 @@ expect.objectContaining = objectContaining;
 expect.arrayContaining = arrayContaining;
 expect.stringMatching = stringMatching;
 
+expect.addSnapshotSerializer = addSerializer;
```

Comment:
This will need to be added in the file that adds the globals in jest-jasmine2. Basically, jest-matchers cannot depend on jest-snapshot. jest-jasmine2 is the extension point :)